spyder/plugins/updatemanager/scripts/install.bat

```diff
@@ -50,10 +51,14 @@ exit %ERRORLEVEL%
     goto :EOF
 
 :update_subroutine
-    echo Updating Spyder
-
-    %conda% update -p %prefix% -y --file %install_file%
-    set /P =Press return to exit...
+    for %%C in ("%install_file%") do set installer_dir=%%~dpC
+    pushd %installer_dir%
+    tar -xf %install_file%
```

Comment:
Is `tar` available by default on Windows? Or does this executable come from a conda package?

Reply:
`tar` is available be default on Windows 10 or greater. Do we need to support older Windows versions?